### PR TITLE
Make finality warnings about our assumptions greyed out

### PR DESCRIPTION
**Pull request by imxeno**

### CODE REVIEW

File: packages/config/src/projects/layer2s/types/Layer2.ts [hunk 1]
```diff
@@ -81,11 +81,16 @@ export interface Layer2LivenessDisplay {
   }
 }
 
+export interface Layer2FinalityDisplayWarning {
+  type: 'project' | 'l2beat'
```
Comment (torztomasz) [inline packages/config/src/projects/layer2s/types/Layer2.ts:85]: I would say it should use `Sentiment` type. `project` and `l2beat` are really abstracted type names. Using `neutral` Sentiment should be good there.

**Review by torztomasz — CHANGES_REQUESTED** [review]
Isn't that PR about finality? Title claims its about liveness

**Comment by imxeno** [comment]
> Isn't that PR about finality? Title claims its about liveness

Oops, yes, it's about finality warnings. Sorry!

File: packages/config/src/projects/layer2s/types/Layer2.ts (hunk 1 above)
Comment (imxeno) [reply packages/config/src/projects/layer2s/types/Layer2.ts]: I've changed the type of the warning fields to - already present - `WarningWithSentiment`.

**Review by torztomasz — APPROVED** [review]

**Review by lucadonnoh — CHANGES_REQUESTED** [review]
polygon zkevm should be grey too

**Review by lucadonnoh — APPROVED** [review]

**Pull request merged**